src/Symfony/Component/Form/Extension/Core/Type/MoneyType.php

```diff
@@ -53,11 +54,22 @@ public function configureOptions(OptionsResolver $resolver)
         $resolver->setDefaults(array(
             'scale' => 2,
             'grouping' => false,
+            'rounding_mode' => NumberToLocalizedStringTransformer::ROUND_HALF_UP,
             'divisor' => 1,
             'currency' => 'EUR',
             'compound' => false,
         ));
 
+        $resolver->setAllowedValues('rounding_mode', array(
+            NumberToLocalizedStringTransformer::ROUND_FLOOR,
```

Comment:
Why using ` NumberToLocalizedStringTransformer ` and not ` NumberFormatter ` here ? Adding a dependency to a transformer without using it doesn't seem right.

Reply:
`NumberToLocalizedStringTransformer` is the parent class of the `MoneyToLocalizedStringTransformer` which is used by this form type and that is configured through these options.

Reply:
Then why not ` MoneyToLocalizedStringTransformer::ROUND_FLOOR ` ?